tests/test_bl45p_mo2.py

```diff
@@ -10,40 +10,37 @@
 from autopep8 import fix_code
 from ruamel.yaml import YAML
 
-from tests.test_render import (  # noqa: F401; pylint: disable=unused-variable
-    pmac_classes,
-)
-
-sample_yaml = Path(__file__).parent / "samples" / "yaml"
-
+from ibek.ioc import IOC
 
 BL45P_MO_IOC_02 = fix_code(
-    """pmac(
+    """IOC(
     ioc_name='bl45p-mo-ioc-02',
     description='an example motion ioc for ibek testing',
     entities=(
         pmac.PmacAsynIPPort(
-            type='pmac.PmacAsynIPPort',
             name='BRICK1port',
-            IP='192.168.0.12:1112'),
+            IP='192.168.0.12:1112',
+            type='pmac.PmacAsynIPPort'),
         pmac.Geobrick(
-            type='pmac.Geobrick',
             name='BL45P-MO-BRICK-01',
-            PORT='BRICK1port',
+            PORT=pmac.PmacAsynIPPort(
+                name='BRICK1port',
+                IP='192.168.0.12:1112',
+                type='pmac.PmacAsynIPPort'),
```

Comment:
This is unfortunate, it is actually a reference to the object defined in line 20, but because we just print the object tree we have no way of knowing this in the test.

This is only going to get worse in subsequent tests, does it make sense to delete this test of intermediate products, and just look at the startup script?

Reply:
I would go with deleting the intermediate product tests. They are a pain to maintain and the cli tests give full coverage anyway.

Reply:
Done